src/day8/re_frame/localstorage.cljs

```diff
@@ -0,0 +1,16 @@
+(ns day8.re-frame.trace.localstorage
+  (:require [goog.storage.Storage :as Storage]
+            [goog.storage.mechanism.HTML5LocalStorage :as html5localstore]))
+
+(defn- storage []
+  (let [mech (goog.storage.mechanism.HTML5LocalStorage.)
+        store (goog.storage.Storage. mech)]
+    store))
+
+(defn get! [key]
```

Comment:
It's often good to reuse the name (especially if it's a good one), and then exclude the Clojure one, like https://clojuredocs.org/clojure.core/refer-clojure#example-542692c8c026201cdc326a5f.